website/static/js/pages/shareEmbedConfig.js

```diff
@@ -0,0 +1,7 @@
+var pym = require('pym.js');
+
+var pymParent = new pym.Parent(
+    'share_registration_iframe',
+    'http://localhost:8000/provider_registration/',
```

Comment:
This should be dropped in from mako in set in settings.py
Ideally place it in `ContextVars`